neuro_extras/main.py

```diff
@@ -217,6 +222,54 @@ def image_copy(source: str, destination: str) -> None:
     run_async(_copy_image(source, destination))
 
 
+@storage.command("copy")
+@click.argument("source")
+@click.argument("destination")
+def cluster_copy(source: str, destination: str) -> None:
+    run_async(_copy_storage(source, destination))
+
+
+async def _copy_storage(source: str, destination: str) -> None:
+    src_uri = URL(source)
+    src_cluster = src_uri.host
+    src_path = src_uri.parts[2:]
+
+    dst_uri = URL(destination)
+    dst_cluster = dst_uri.host
+    dst_path = dst_uri.parts[2:]
+
+    assert src_cluster
+    assert dst_cluster
+    async with neuro_api.get() as client:
+        await client.config.switch_cluster(dst_cluster)
+        await client.storage.mkdir(URL("storage:"), parents=True, exist_ok=True)
+    await _run_copy_container(src_cluster, "/".join(src_path), "/".join(dst_path))
+
+
+async def _run_copy_container(
+    src_cluster: str, src_path: str, dst_path: str
+) -> Process:
+    args = [
+        "neuro",
+        "run",
+        "-s",
+        "cpu-small",
+        "--pass-config",
+        "-v",
+        "storage:://storage",
+        "-e",
+        f"NEURO_CLUSTER={src_cluster}",
+        "neuromation/neuro-extras:latest",
+        f'"cp -r -u -T storage:{src_path} /storage/{dst_path}"',
+    ]
+    cmd = " ".join(args)
+    subprocess = await asyncio.create_subprocess_shell(cmd)
```

Comment:
I think, it's better to just print the result command to stdout not to execute it, similar to what we for seldon:
```
$ nuero-extras cluster-cp storage:neuro-ai-public/* storage:neuro-public/* | bash
```
Or at least let's print this command before executing it, for transparency.

Reply:
also, a unit test would be enough for that.

Reply:
I've added printing the command.